workspace-manager-clienttests/README.md

```diff
@@ -49,6 +49,15 @@ Perf Test
 ```
 ./gradlew  runTest --args="suites/BasicPerf.json /tmp/TR"
 ```
+#### Upload test results to Google Bucket
+To upload Test Runner results to a Google Bucket.
+The bucket location is specified as a parameter in an upload config file located in the `resources/uploadlists` directory
```

Comment:
nit: I'd probably have looked in the GHA workflow for a parameter for this location, so the workflow is more reusable.

Reply:
This will be a future enhancement.